spec/lib/nerve_spec.rb

```diff
@@ -0,0 +1,186 @@
+require 'spec_helper'
+require 'nerve/configuration_manager'
+require 'nerve/service_watcher'
+require 'nerve/reporter'
+require 'nerve/reporter/base'
+require 'nerve'
+
+def make_mock_service_watcher
+    mock_service_watcher = double(Nerve::ServiceWatcher)
```

Comment:
We are verifying doubles, e.g. if I say "allow(mock_service_watcher).to receive(:nope).and_return(true)" rspec gets mad at me with "expected no Exception, got #<RSpec::Mocks::MockExpectationError: Nerve::ServiceWatcher does not implement: nope>".

Apparently 'just' making it a double or instance_double doesn't allow it to receive expected method calls (this seems contrary to what the documentation says, but go ahead and try it).